partner_title/models/res_partner.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2025 Akretion France (https://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    title_id = fields.Many2one(
```

Comment:
I think the field name should be changed to `title` following 18.0, or a migration script needs to be added?